### ISSUE TITLE
turn cost investigation bullet point from https://github.com/sul-dlss/speech-to-text/issues/3 into separate ticket, see if back of the envelope estimates could rule out some of our possible implementations, or if they all look close enough in cost to keep pursuing all paths (ECS, always on EC2 VM, thin vs heavy whisper docker container, etc).  cost calculator vs deploy and test w/ tagged resources?

### ISSUE THREAD

**Issue by jmartin-sul** [sul-dlss/speech-to-text#19]

**Comment by jmartin-sul** [comment]
@edsu feel free to claim this and put it in "in progress" if you're already working on it.  anyone else, feel free to also assign yourself simultaneously and coordinate, if you're exploring paths besides tagged deployment of a docker container (or @edsu can say if he'd like to claim this whole issue for now, to prevent duplicated effort)

**Comment by jmartin-sul** [comment]
we might need a bit more of a handle on what ECS deployment and scaling look like, before we can really measure cost.

but we can also get a sense of how much it costs to caption a certain duration of media in ECS, and same in EC2.  might be able to get a rough sense of cost by measuring that in the absence of ability to scale.

first few months might see the service on all the time anyway, since we probably have a substantial backlog to caption.

@andrewjbtw is interested in unit cost, and using that to determine how much we need to throttle e.g. bulk actions or preassembly batches for captioning.  throttling may come naturally from our ECS or EC2 setup?  e.g. set a limit on the number of STT containers, which are essentially single threaded (poll, work, poll, work, ...).
